### PR TITLE
Setup Bintray publication

### LINKED ISSUE #95: Setup publication to Bintray

### PR DESCRIPTION
**Pull request by arkivanov**
Fixes #95

### CODE REVIEW

File: publication.gradle [hunk 1]
```diff
@@ -29,3 +29,29 @@ kotlin {
         }
     }
 }
+
+apply plugin: 'com.jfrog.bintray'
+
+bintray {
+    user = findProperty('user')
```
Comment (CherryPerry) [inline publication.gradle]: Rename please. `bintray.name` or something like that.

File: publication.gradle [hunk 2]
```diff
@@ -29,3 +29,29 @@ kotlin {
...
+        name = 'reaktive'
+        userOrg = 'badoo'
+        vcsUrl = 'https://github.com/badoo/Reaktive.git'
+        licenses = ['Apache-2.0']
+        version.name = reaktive_version
+    }
+}
+
+// This is required for K/N publishing
+bintrayUpload.dependsOn publishToMavenLocal
```
Comment (CherryPerry) [inline publication.gradle]: Can you use [Tasks Provider API](https://docs.gradle.org/current/userguide/task_configuration_avoidance.html)? `tasks.getByName('bintrayUpload') { dependsOn tasks.getByName('publishToMavenLocal') }`. Same for next task.

**Comment by CherryPerry** [comment]
Don't try to release now, just merge. Travis does not know about `bintrayUpload`, user name and API key.

**Comment by arkivanov** [comment]
What does it mean? How is it connected with Travis?

**Comment by CherryPerry** [comment]
Well, you want to release (upload) builds with help of Travis, but now it can't.

**Comment by arkivanov** [comment]
I thought it should be a completely manual job. When you want to release then just execute the Gradle task with your own credentials.

**Comment by CherryPerry** [comment]
Why should it not be automatically released when you add tag? Same as Jitpack now.
We can set API key for Bintray as enviroment variable on TravisCI.

And do you have MacOS to release it by yourself :) ?

**Comment by arkivanov** [comment]
Well this should be done separately and sounds like it does not prevent us from releasing. Anyway I'm waiting for https://github.com/badoo/Reaktive/pull/96 to be merged.

**Pull request merged**